bahmnicore-api/src/test/java/org/bahmni/module/bahmnicore/dao/impl/VisitDaoImplIT.java

```diff
@@ -50,10 +50,47 @@ public void shouldNotGetVoidedEncounter() throws Exception {
     }
 
     @Test
-    public void shouldReturnAllEncountersWhenNumberOfVisitsAndEncountersAreNotProvided() {
+    public void shouldReturnAllNonVoidedEncountersWhenNumberOfVisitsAndEncountersAreNotProvided() {
```

Comment:
@shivarach what do you mean by NonVoidedEncounters ??

Reply:
Encounters that are not voided. The query specifically looks for non-voided encounters.